docs/modules/ROOT/pages/index.adoc

```diff
@@ -428,7 +448,109 @@ The signature type is represented by the `sig_type` field and must be one of the
+
+The assertion label for such an assertion MUST be `cawg.identity.vc`.
+
+The _<<_credential_holder,credential holder’s>>_ _<<W3C verifiable credential>>_ is not itself directly embedded in the *<<_identity_assertion,identity assertion>>,* but is instead used to issue a new _<<W3C verifiable credential>>_ describing the _<<_credential_subject,credential subject’s>>_ role in the specific _<<C2PA asset>>_ being described.
+
+The relationships among _<<_credential_holder,credential holder’s>>_ _<<W3C verifiable credential>>,_ _<<W3C decentralized identifier document>>,_ *identity assertion’s* _<<W3C verifiable credential>>,_ the `tbs` data structure, and the _<<_c2pa_asset,C2PA asset>>_ are illustrated in the following diagram, which is non-normative:
+
+[mermaid,width=100%]
+....
+erDiagram
+  CH["Credential holder"] {}
+  CHVC["Credential holder’s verifiable credential"] {}
+  TBS["`tbs` data structure"] {}
+  IVC["Identity assertion’s verifiable credential"] {}
+  ASSET["C2PA asset"] {}
+
+  CH ||--|| CHVC : "controls via private key"
+  IVC||--|| TBS: "references"
+  IVC ||--|| CHVC: "issued by"
+  TBS ||--|| ASSET: "describes"
+....
+
+==== Credential holder’s verifiable credential
+
+TO DO: Specify requirements for CH VC.
+
+* Needs to have discoverable `assertionMethod`
+* DID method recommendations/requirements?
+* Alignment with ToIP verifiable identifier
+
+==== Identity assertion’s verifiable credential
+
+The _<<_c2pa_claim_generator,C2PA claim generator>>_ MUST generate the `tbs` data structure as described in *TO DO: Xref tbs once merged* and present that to the _<<_credential_holder,credential holder>>_ for signature.
+
+The _<<_credential_holder,credential holder>>_ MUST issue a _new_ _<<_w3c_verifiable_credential,W3C verifiable credential>>_ which satisfies the following requirements:
+
+* The credential MUST meet all requirements for a verifiable credential as described in https://www.w3.org/TR/vc-data-model-2.0/[Verifiable Credentials Data Model, version 2.0].
+* The `@context` field MUST contain at least the following entries:
+** `"https://www.w3.org/ns/credentials/v2"`
+** `"https://creator-assertions.github.io/tbd/tbd"`
+* The `type` field MUST contain at least the following entries:
+** `"VerifiableCredential"`
+** `"CreatorIdentityAssertion"`
+* The `credentialSubject` field MUST contain a `tbs` entry, which MUST be the JSON serialization of the `tbs` data structure presented for signature. All bytestring entries in the `tbs` data structure (i.e. `hash` entries in hashlink URL data structures) MUST be converted to the corresponding Base-64 plain-text strings. (TO DO: Specify Base-64 encoding options.)
+* The `credentialSubject` field MAY contain other entries. The meaning of any such entries is not defined in this specification.
+* The `issuer` field MUST exist.
+* A suitable `proof` field MUST exist and prove issuance of this credential by `issuer`. (TO DO: Review https://www.w3.org/TR/vc-data-integrity/.)
```

Comment:
In issue #73 there is an ongoing discussion what security recommendations could be. As per [4.9 Securing Mechanisms](https://www.w3.org/TR/vc-data-model-2.0/#securing-mechanisms) section, there are to options:
- enveloping proof ([DATA INTEGRITY](https://www.w3.org/TR/vc-data-integrity/))
- enveloping proof ([VC-JOSE-COSE](https://www.w3.org/TR/vc-data-model-2.0/#bib-vc-jose-cose))